pkgs/development/libraries/libfilezilla/default.nix

```diff
@@ -31,6 +31,7 @@ stdenv.mkDerivation rec {
     description = "A modern C++ library, offering some basic functionality to build high-performing, platform-independent programs";
     license = licenses.gpl2Plus;
     maintainers = with maintainers; [ pSub ];
-    platforms = platforms.unix;
+    # darwin requires apple_sdk >= 10.13
+    platforms = lib.platforms.linux;
```

Comment:
We have `apple_sdk_11_0` so it should be manageable.

Reply:
Indeed, the lib now works but I'm still trying to compile filezilla on macos